devel/git/Portfile

```diff
@@ -209,7 +209,7 @@ variant credential_osxkeychain description {Install git credential-osxkeychain u
 
     pre-destroot {
         xinstall -m 755 "${worksrcpath}/contrib/credential/osxkeychain/git-credential-osxkeychain" \
-            "${destroot}${prefix}/bin/"
+            "${destroot}${prefix}/libexec/git-core/"
```

Comment:
git installation creates this directory. I believe we can just move this into post-destroot. See https://github.com/macports/macports-ports/pull/291